src/Fields/KeyValueField.php

```diff
@@ -80,11 +80,8 @@ public function __construct($name, $title = null, $sourceKeys = [], $sourceValue
 
     public function Field($properties = [])
     {
-        if (Controller::curr() instanceof ContentController) {
-            Requirements::javascript('silverstripe/admin: thirdparty/jquery/jquery.js');
-        }
```

Comment:
That's not stored there anymore - and even if it was, we shouldn't just inject jQuery into peoples' projects like this